contracts/access/AccessController.sol

```diff
@@ -164,21 +146,22 @@ contract AccessController is IAccessController, AccessManagedUpgradeable, UUPSUp
         // This includes initiating calls to these modules and receiving calls from them.
         // Additionally, it can modify Permissions settings.
         AccessControllerStorage storage $ = _getAccessControllerStorage();
-        if (
-            to != address(this) &&
-            !IModuleRegistry($.moduleRegistry).isRegistered(to) &&
-            !IModuleRegistry($.moduleRegistry).isRegistered(signer)
-        ) {
-            revert Errors.AccessController__BothCallerAndRecipientAreNotRegisteredModule(signer, to);
-        }
         // Must be a valid IPAccount
         if (!IIPAccountRegistry($.ipAccountRegistry).isIpAccount(ipAccount)) {
             revert Errors.AccessController__IPAccountIsNotValid(ipAccount);
         }
-        // Owner can call all functions of all modules
+        // Owner can call any contracts either registered module or unregistered/external contracts
```

Comment:
Need to revisit any risk of opening up IPAccounts to call external contracts freely (previously it was through "external" modules).

Is the "pass-through" risk discussed some time ago still relevant in this context? cc @Spablob @Ramarti

Reply:
It is only the IP owner can call external contracts.  no change to the behavior of "pass-through."

Reply:
That comment states that the owner can call anything through the IPA, which is the expected outcome of ERC6551

Reply:
right, ERC6551 doesn't restrict target of execute() of IPAccount.